apps/test/unit/lib/kits/maker/ui/SetupChecklistTest.js

```diff
@@ -135,16 +124,27 @@ describe('SetupChecklist', () => {
  * @param {number} intervalMs - time to wait between steps
  * @return {Promise}
  */
-function yieldUntil(wrapper, predicate, timeoutMs = 3500, intervalMs = 5) {
+function yieldUntil(
```

Comment:
nit: not a part of this pr probably, but rather a general comment. I wonder why this function here is declared in the bottom of the file instead of the top section since it's used in tests. I mean, of course, it work even now, but it's harder to read it this way I think

Reply:
Yeah - I see this also in `SetupGuideTest.js`. I'm not sure what the standard convention is. I actually prefer the function definitions at the bottom, but I can ask the group if there's a general preference during check-in. Thanks!